mysql-compatibility.md

```diff
@@ -21,6 +21,7 @@ However, some features of MySQL are not supported. This could be because there i
 + Events
 + User-defined functions
 + `FOREIGN KEY` constraints [#18209](https://github.com/pingcap/tidb/issues/18209)
++ Temporary tables
```

Comment:
This is already mentioned below as `CREATE TEMPORARY TABLE syntax #1248`.

Reply:
@pepezzzz This is duplicate with [Line 35](https://github.com/pingcap/docs/pull/3796/files#diff-7028919502ad5a35697a652366fdafadR35). What about removing it?

Reply:
Like Forgien Key feature, CREATE TEMPORARY TABLE syntax is compatitable, but TiDB create a real table insteadly. I suggest that  add the issue link after TEMPORARY TABLE feature, and remove the syntax line, keepace with Forgien Key.

Reply:
SGTM

Reply:
@pepezzzz done in https://github.com/pingcap/docs/pull/3796/commits/6078be93bebd7b847b1a46cf2e7df4bb31408287. PTAL